assignments/pig-latin/code.js

```diff
@@ -31,3 +31,15 @@
 //
 // Feel free to play around in the REPL to make sure you understand how this
 // operator works.
+
+
+const pigLatin = (s) => (s.substring(s.search(/[aeiou]/), s.length)+s.substring(0,s.search(/[aeiou]/))+'ay')
```

Comment:
You don't need the `()`s around the expression. So this is fine:

```suggestion
const pigLatin = (s) => s.substring(s.search(/[aeiou]/), s.length)+s.substring(0,s.search(/[aeiou]/))+'ay';
```

But this function would also benefit from using a variable like you did in `advancedPigLatin`. Something like

```javascript
const pigLatin = (s) => {
  const i = s.search(/[aeiou]/);
  return s.substring(i) + s.substring(0, i) + 'ay';
};
```

This saves a tiny bit of work for the computer (it only has to search the string once) but more importantly makes everything easier to read because you can understand the first line on its own (ah, we're getting the index of the first vowel) and then read the final expression with that in mind rather than having to understand both things at the same time. Also it's just easier to read and see that the `i` is the same in both places rather than the more complicated `search` expression.

Finally, you should use the one-arg version of `substring` for the first call just because it makes it easier to read at a glance and see that it's going all the way to the end of the string.